app/templates/detail.hbs

```diff
@@ -48,64 +48,42 @@
       </c.header>
       <c.content>
 
-        {{#if this.model.session.approvedDate}}
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
-            <AuIcon
-              class="au-u-margin-right-small"
-              @size="large"
-              @icon="check"
-            />
-            <p>
-              Goedgekeurd -
-              {{iso-formatter this.model.session.approvedDate}}
-
-            </p>
-          </section>
-        {{/if}}
-
-        {{#if this.model.municipality}}
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
-            <AuIcon
-              class="au-u-margin-right-small"
-              @size="large"
-              @icon="check"
-            />
-            <p>{{this.model.title_agenda.value}}
-              -
-              {{#if this.model.geplandeStart.value}}
-                {{iso-formatter this.model.geplandeStart.value}}
-              {{else}}
-                Geen Datum
-              {{/if}}</p>
-          </section>
-          <section
-            class="au-u-flex au-u-muted au-u-margin-top-tiny"
-            style=" align-items: center;"
-          >
+        <section
+          class="au-u-flex au-u-muted au-u-margin-top-tiny"
+          style=" align-items: center;"
+        >
+          <AuIcon class="au-u-margin-right-small" @size="large" @icon="check" />
+          <p>
+            {{#if this.model.session.startedAt}}
+              {{iso-formatter this.model.session.startedAt}}
+            {{else}}
+              Geen Datum
+            {{/if}}</p>
+        </section>
+        <section
+          class="au-u-flex au-u-muted au-u-margin-top-tiny"
+          style=" align-items: center;"
+        >
+          {{#if this.model.session.governingBody.administrativeUnit.name}}
```

Comment:
@aliokan how would you want me to display the links in case there is no municipality? 
Do I check on whether it is equal to the value of "Ontbrekende bestuurseenheid" in an if statement?

Reply:
You right. We can keep the `{{#if this.model.session.governingBody.administrativeUnit.name}}`. Or add a hasMunicipality directly on session that return a boolean